core/services/aws/commons/ddb/LocationDynamoDbOperations.ts

```diff
@@ -0,0 +1,37 @@
+import DynamoDbTableOperations from './DynamoDbTableOperations'
+import {
+  QueryConditionOperator,
+  QueryInput
+} from '../../../../application/models/policies/repositories/QueryTypes'
+import { LocationDTO } from 'commons/dto/UserDTO'
+import LocationModel, { LocationFields } from 'core/application/models/dbModels/LocationModel'
+
+export default class LocationDynamoDbOperations extends DynamoDbTableOperations<
+LocationDTO,
+LocationFields,
+LocationModel
+> {
+  async deleteUserLocations(userId: string): Promise<void> {
+    const primaryIndex = this.modelAdapter.getPrimaryIndex()
+    const query: QueryInput<LocationFields> = {
+      partitionKeyCondition: {
+        attributeName: primaryIndex.partitionKey,
+        operator: QueryConditionOperator.EQUALS,
+        attributeValue: `USER#${userId}`
+      }
+    }
+
+    if (primaryIndex.sortKey != null) {
```

Comment:
as we are using strict equals, shouldn't this be `===` ?

Reply:
updated in fb88d811b0c2627d95a87ab81566224901d34298